static/search.js

```diff
@@ -0,0 +1,307 @@
+          }
+
+          update();
+        }).catch(function(err){
+          alert("Error:" + err);
+          clearInterval(load);
+        });
+}
+
+/**
+ * This function will give the result of the input query.
+ * @param {string} query The input cypher query
+ */
+function query_update(query) {
+  // MATCH (a)-[b]->(c)-[d]->(e)-[f]->(g) WHERE a.id = 'SWARCH_MEMORY-ARCHITECTURE_DIAGRAM' RETURN a,b,c,d,e,f,g
+  graph.nodes = [];
+  graph.links = [];
+  selectedNodeID = undefined;
+  const session = driver.session({database:"neo4j"});
+  session.run(query)
+        .then(function(result){
+          var nodes_made = [];
+          for (var i = 0; i < result.records.length; i++){
+            var types = result.records[i]._fields; //node relation node
+            for (var type in types){
+              if (types[type].__isNode__){
+                if (!nodes_made.includes(types[type].properties["id"])){
+                  graph.nodes.push(types[type].properties);
+                  nodes_made.push(types[type].properties["id"]);
+                }
+              }
+              if (types[type].__isRelationship__){
+                graph.links.push({source: types[type].properties.source, target: types[type].properties.target, label: types[type].type, colour: types[type].properties.colour})
+              }
+              if (types[type].__isPath__){
+                console.log("path")
+                for (var pathsegment in types[type].segments){
+                  for (var t in types[type].segments[pathsegment]){
+                    if (types[type].segments[pathsegment][t].__isNode__){
+                      if (!nodes_made.includes(types[type].segments[pathsegment][t].properties["id"])){
+                        graph.nodes.push(types[type].segments[pathsegment][t].properties);
+                        nodes_made.push(types[type].segments[pathsegment][t].properties["id"]);
+                      }
+                    }
+                    if (types[type].segments[pathsegment][t].__isRelationship__){
+                      graph.links.push({source: types[type].segments[pathsegment][t].properties.source, target: types[type].segments[pathsegment][t].properties.target, label: types[type].segments[pathsegment][t].type, colour: types[type].segments[pathsegment][t].properties.colour})
+                    }
+                  }
+                }
+              }
+            }
+          }
+          if (graph.nodes.length === 0 && graph.links.length === 0){
+            alert("This query gives empty data. Try another query. \nNote that names are case sensitive.");
+          }
+          console.log("data loaded");
+          update();
+        })
+        .catch(function(err){
+          alert("Error:" + err);
+          clearInterval(load);
+        });
+}
+
+
+
+/**
+ * This function will update the graph with the new nodes and links. It will restart the simulation.
```

Comment:
```suggestion
 * Updates the nodes and links of the graph and restarts the simulation.
```
Please update every docstring to use this style of wording.